pkg/v1/tkg/clusterclient/clusterclient.go

```diff
@@ -860,6 +860,15 @@ func (c *client) waitK8sVersionUpdateGeneric(clusterName, namespace, newK8sVersi
 		return false, err
 	}
 
+	// Wait for 90 seconds before we start the polling to check upgrade status
+	// This is done to account for the CP/MD upgrade to start rolling out
+	// because it was noticed that CP/MD upgrade rollout take few seconds
+	// to update the conditions which we rely on to check the upgrade status
+	// ClusterReady is already present and wait logic comes out of the poll
+	// loop assuming the upgrade is complete whereas the upgrade has not been
+	// started yet
+	time.Sleep(90 * time.Second)
```

Comment:
good to define a constant for the wait duration. 

This means we are applying a wait before checking on both the control plane and worker nodes. But given the current state is pretty harmless to do.